### PR TITLE
Xentriq 1.0 documentation

### PR DESCRIPTION
**Pull request by mpwaldhorst**

### CODE REVIEW

File: docs/templates/tags/conditionals.rst [hunk 1]
```diff
@@ -62,3 +62,40 @@ Currently the following is not supported:
 - Calculations in conditions such as division, multiplication, addition, substraction, modulo
   
   - Example: [if var-1 == 0] 
+
+Scope
+-----
+Conditional nodes creates their own local scope.
```
Comment (mpwaldhorst) [inline docs/templates/tags/conditionals.rst]: Typo, should be `create`

File: docs/templates/tags/include-implement.rst [hunk 2]
```diff
@@ -1,16 +1,11 @@
-Include & Implement
```
Comment (mpwaldhorst) [inline docs/templates/tags/include-implement.rst]: file should be renamed.

File: docs/templates/tags/include-implement.rst [hunk 3]
```diff
@@ -48,11 +37,11 @@ And the template that should implement the layout looks like this:
 
 .. code-block:: html
 
-      [implement:'wrapper']
+      [insert:'wrapper']
         [part:'default']
```
Comment (mpwaldhorst) [inline docs/templates/tags/include-implement.rst]: does it make sense to rename part to section? 🤔

File: docs/templates/tags/include-implement.rst [hunk 4]
```diff
@@ -63,30 +52,83 @@ The resulting rendered template will be:
...
+   :caption: insert.xpart
+
+    Hello there.
+
+    [part='name' /]
+
+.. code-block:: html
+   :caption: template.xpage
+
+    [name=force->getOpponent /] <!-- Assume this is set to `General Kenobi` -->
```
Comment (mpwaldhorst) [inline docs/templates/tags/include-implement.rst]: Maybe add here that this variable is not available in insert.xpart if not passed through the tag.

File: docs/templates/tags/include-implement.rst [hunk 5]
```diff
@@ -63,30 +52,83 @@ The resulting rendered template will be:
...
+
+    [insert:'insert-template']
+      [part:'name']
+        [name] <!-- Read from template scope -->
```
Comment (mpwaldhorst) [inline docs/templates/tags/include-implement.rst]: Conversely mention that the variable is available here but not implicitly in the insert.xpart

File: docs/templates/tags/link.rst [hunk 6]
```diff
@@ -1,18 +1,35 @@
...
 ====
 
-By using the link-tag it is possible to retrieve the current url (e.g. http://localhost/current/path)
+By using the link-tag it is possible to retrieve the current url, root url or generate a link to another template based on a path rooted at the templates directory (e.g. http://localhost/path)
 
+For more details about project layout see :doc:`/projects/project-structure`
+
+Scope
+-----
+Single tag only, has no effect on active scope. Can be assigned to variable in current scope or output immediately.
```
Comment (mpwaldhorst) [inline docs/templates/tags/link.rst]: what is the difference between active and current scope. Maybe it should be only affects current scope if assigned to variable?

File: docs/templates/tags/variable.rst [hunk 7]
```diff
@@ -1,15 +1,16 @@
...
 
-You can apply an argument or modifier to a variable to format the output;
+Variables in Xentriq are represented the same as some of the other tags. 
 
-.. code-block:: html
+Modifiers
+---------
 
-      [var <modifier1>="<modifier1-value>" <modifier2>]
+You can apply a modifier to a variable to format or change the output;
```
Comment (mpwaldhorst) [inline docs/templates/tags/variable.rst]: why a semi-colon?

File: docs/templates/tags/include-implement.rst (hunk 2 above)
Comment (ttveldhuis) [reply docs/templates/tags/include-implement.rst]: Agreed

File: docs/templates/tags/include-implement.rst (hunk 3 above)
Comment (ttveldhuis) [reply docs/templates/tags/include-implement.rst]: Possibly. Maybe `fragment`?

File: docs/templates/tags/link.rst (hunk 6 above)
Comment (ttveldhuis) [reply docs/templates/tags/link.rst]: Current scope and active scope are different names for the same thing. We should probably choose one of each. The first sentence is a bit vague, but I meant that the current scope doesn't change to a new scope.

File: docs/templates/tags/variable.rst (hunk 7 above)
Comment (ttveldhuis) [reply docs/templates/tags/variable.rst]: I think the there should be no type of colon at all :p

File: docs/templates/tags/include-implement.rst (hunk 4 above)
Comment (ttveldhuis) [reply docs/templates/tags/include-implement.rst]: I have made a change to this, could you review?

File: docs/templates/tags/include-implement.rst (hunk 5 above)
Comment (ttveldhuis) [reply docs/templates/tags/include-implement.rst]: I have made a change to this, could you review?

File: docs/templates/tags/link.rst (hunk 6 above)
Comment (ttveldhuis) [reply docs/templates/tags/link.rst]: I have made a change to this, could you review?

**Pull request merged**